blueprints/resilient-apps-multiple-machines.html.md

```diff
@@ -94,7 +94,7 @@ To create a standby Machine from a different image:
 fly machine run <image> --standby-for <machine-id>
 ```
 
-## Benefits and cost of multiple Machines
+## Benefits and costs of multiple Machines
```

Comment:
leave as-is, since "Cost" is use below and is the intended word here together with "of"

Reply:
I have made the required change. Thank you for the review.